From: Daniel Wagner <dwagner@suse.de>
Date: Tue, 31 May 2022 15:12:11 +0200
Subject: lpfc: Set default protocol support to FCP only
Patch-mainline: Never, restore default settings
References: bsc#1194124 bsc#1198899

Several regression have been reported due to enabling NVME support
per default. The default was changed during the driver update
and hence it differs in behavior compared with initial release of
SLE12-SP5. Let's set the default back to FCP only.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/lpfc/lpfc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/lpfc/lpfc.h b/drivers/scsi/lpfc/lpfc.h
index 669f30cde0e6..c2dbb0787ddf 100644
--- a/drivers/scsi/lpfc/lpfc.h
+++ b/drivers/scsi/lpfc/lpfc.h
@@ -1075,7 +1075,7 @@ struct lpfc_hba {
 #define LPFC_ENABLE_BOTH 3
 #if (IS_ENABLED(CONFIG_NVME_FC))
 #define LPFC_MAX_ENBL_FC4_TYPE LPFC_ENABLE_BOTH
-#define LPFC_DEF_ENBL_FC4_TYPE LPFC_ENABLE_BOTH
+#define LPFC_DEF_ENBL_FC4_TYPE LPFC_ENABLE_FCP
 #else
 #define LPFC_MAX_ENBL_FC4_TYPE LPFC_ENABLE_FCP
 #define LPFC_DEF_ENBL_FC4_TYPE LPFC_ENABLE_FCP
-- 
2.29.2

